tests/is-winner/pass_kigiri.js

```diff
@@ -0,0 +1,22 @@
+const isWinner = async (name) => {
```

Comment:
I did 2 solutions, just so you can compare using `async / await` and `.then / .catch`